.github/workflows/sei-wasmd-unit_tests.yml

```diff
@@ -0,0 +1,87 @@
+name: sei-wasmd 
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
nit: should we make this run only on specific paths as well, so we don't run irrelevant CI on unrelated changes?
eg.

```
    paths:
      - 'sei-wasmd/**'
```